scripts/run_stress_tests.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+
+CWD=$(pwd)
+BIN="$CWD/../release"
+SCRIPTS="$CWD"
+#MOUNT_OPTS="-t famfs -o noatime -o dax=always "
+TEST_ALL=1
+SLEEP_TIME=2
+
+
+# Allow these variables to be set from the environment
+if [ -z "$MPT" ]; then
+    MPT=/mnt/famfs
+fi
+if [ -z "$DEV" ]; then
+    DEV="/dev/pmem0"
+fi
+
+# Verify that this script is running as root
```

Comment:
This script doesn't need to be run as root, but it does need sudo (which is the next test). So drop this test.

Reply:
Ok removed.